src/components/Nav.tsx

```diff
@@ -0,0 +1,35 @@
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Nav: React.FC = () => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            to={'/'}
+            className={({ isActive }) =>
+              cn('navbar-item', { 'has-background-grey-lighter': isActive })
+            }
```

Comment:
Would be good to create separate function for `({ isActive }) =>
              cn('navbar-item', { 'has-background-grey-lighter': isActive })
            }` and reuse it here and below